src/components/Actions/actionTypes.js

```diff
@@ -0,0 +1,17 @@
+export const FETCH_CAR = 'CAR_FETCH_CAR';
+export const FETCH_CAR_SUCCESS = 'CAR_FETCH_CAR_SUCCESS';
+export const FETCH_CAR_FAILURE = 'CAR_FETCH_CAR_FAILURE';
+
+export const FETCH_AUTENTICATION = 'AUTENTICATION_FETCH_AUTENTICATION';
+export const FETCH_AUTENTICATION_SUCCESS =
+  'AUTENTICATION_FETCH_AUTENTICATION_SUCCESS';
+export const FETCH_AUTENTICATION_FAILURE =
+  'AUTENTICATION_FETCH_AUTENTICATION_FAILURE';
+
+export const FETCH_USERS = 'FETCH_USERS';
```

Comment:
You should stick to a consistent naming rule, and just FYI these constants can't collide with one another so a complex and specific naming rule wouldn't be a bad idea. ( Rule as in `rule of thumb`)